src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+public class RightTriangle extends Figure {
+    private final double firstLeg;
+    private final double secondLeg;
+
+    public RightTriangle(Color color, double firstLeg, double secondLeg) {
+        super(color, (firstLeg * secondLeg) / 2);
```

Comment:
Creating a figure, don't pass expressions in the constructor. Instead, create separate variables for the area calculation and pass them on for better code readability.